### DISCUSSION TITLE
Proxmox 8 compatibility

### DISCUSSION THREAD

**Discussion by Pete1979** [TheGrandWazoo/freenas-proxmox#145] [General]
Hi,

Has this been tested on Proxmox 8?

BR
Peter

**Comment by awptechnologies** [comment]
I have proxmox 8 installed with 0 issues. Only thing i did was keep current version of sshd instead of going with the new package maintainers version. Only thing is i get an error in proxmox gui because the freenas-proxmox repository is still bullseye. I tried to switch the repository to bookworm and it was unable to connect. Once the creator makes it available in the bookworm repository it should have no errors at all.

**Reply by gionag** [reply]
me too. works great on 8
